macros/on-run-end/print_dbt_project_evaluator_issues.sql

```diff
@@ -0,0 +1,51 @@
+{% macro _return_list_header_rows(query) %}
+
+    {%- call statement('get_query_results', fetch_result=True,auto_begin=false) -%}
+
+        {{ query }}
+
+    {%- endcall -%}
+
+    {% set sql_results=[] %}
+
+    {%- if execute -%}
+        {% set sql_results_table = load_result('get_query_results').table %}
+        {% do sql_results.append(sql_results_table.column_names) %}
+        {% for row_data in sql_results_table.rows %}
+            {% do sql_results.append(row_data.values()) %}
+        {% endfor %}
+    {%- endif -%}
+
+    {{ return(sql_results) }}
+
+{% endmacro %}
+
+
+{% macro print_dbt_project_evaluator_issues(schema_package=None) %}
+
+  {{ print("\n### List of issues raised by dbt_project_evaluator ###") }}
+
+  {% for result in results | selectattr('failures') | selectattr('failures', '>', 0) %}
+    
+    {% if result.node.fqn[0] == "dbt_project_evaluator" %}
+      
+      {{ print("\n-- " ~ result.node.alias ~ " --") }}
+
+      {% set model_checked = result.node.depends_on.nodes[0].split('.')[-1] %}
+
+      {% set sql_statement %}
+      select * from {{ schema_package ~ "." ~  model_checked if schema_package else model_checked }}
```

Comment:
Could they also have this package's models in a custom database?

Reply:
They could, and I can add it, but I also want to avoid making it capable of handling all situations. If some people don't like the current implementation they can also get inspired by this and create their own macros.